src/main/kotlin/app/CacheService.kt

```diff
@@ -0,0 +1,84 @@
+@file:Suppress("SqlResolve")
+
+package app
+
+import app.util.HikariCpDataSource
+import com.fasterxml.jackson.module.kotlin.jacksonObjectMapper
+import com.fasterxml.jackson.module.kotlin.readValue
+import org.slf4j.LoggerFactory
+import java.time.Instant
+import java.time.temporal.ChronoUnit
+
+object CacheService {
+    private val log = LoggerFactory.getLogger(CacheService.javaClass)
+    private val objectMapper = jacksonObjectMapper()
+
+    private fun createTableIfAbsent() {
+        val connection = HikariCpDataSource.connection
+        val statement = connection.createStatement()
```

Comment:
Why not just 

```
val statement = HikariCpDataSource.connection.createStatement()
```

Reply:
https://github.com/Attacktive/profile-summary-for-github/commit/131228688dc8248a946f98a7c8067cbb6c6a65b0